<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElevatorManager</class>
 <widget class="QMainWindow" name="ElevatorManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ElevatorManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>611</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="elevatorsView"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>451</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="buttonsView"/>
   </widget>
   <widget class="QSpinBox" name="minFloorInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxFloorInput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addElevatorButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>520</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add elevator</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>20</width>
      <height>491</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>643</x>
      <y>10</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator buttons</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator visualisation</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="settingsButton"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="settingsButton">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
